app/templates/events/view/speakers.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="ui grid stackable">
     {{#if (and (not-eq this.session.currentRouteName 'events.view.speakers.edit') (not-eq this.session.currentRouteName 'events.view.speakers.create'))}}
       <div class="row">
-        <div class="six wide column">
+        <div class="{{if this.device.isTablet 'sixteen' 'twelve'}} wide column">
```

Comment:
This is duplicate of #5907, check the comments there

Reply:
ok